docker/dev/config/powermon.yaml

```diff
@@ -1,6 +1,8 @@
 # yaml config for powermon
```

Comment:
i assume this is the 'current' working config file - think we need to delete the incorrect ones